kf_lib_data_ingest/app/cli.py

```diff
@@ -18,6 +19,25 @@
 
 CONTEXT_SETTINGS = {"help_option_names": ["-h", "--help"]}
 DEFAULT_LOG_LEVEL_NAME = logging._levelToName.get(DEFAULT_LOG_LEVEL)
+DEFAULT_VALIDATION_MODE = VALIDATION_MODES[1]
```

Comment:
If the number of modes change, this will get weird. Can the modes be given their own constants and then use the right one directly?